docs/modules/clients/pages/clc.adoc

```diff
@@ -0,0 +1,350 @@
+= Hazelcast Command-Line Client
```

Comment:
How about "Command Line Client" instead of "Hazelcast Command-Line Client" ?

Reply:
I think the existing title is good. Users will likely search Google for the Hazelcast CLC so having Hazelcast in the title is helpful.